spec/sign_up_spec.rb

```diff
@@ -0,0 +1,82 @@
+feature 'Signing up to Chitter' do
+
+  let!(:user) do
+    User.new(name: 'Katie', username: 'Smithy', email: 'K@email.com', password: 'secret', password_confirmation: 'secret')
+  end
+
+  let!(:user2) do
+    User.new(name: 'Jack', username: 'Jacko', email: 'J@email.com', password: 'blue', password_confirmation: 'blue')
```

Comment:
Metrics/LineLength: Line is too long. [116/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)